bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/RuleEnableModuleType.json

```diff
@@ -0,0 +1,26 @@
+{
+	"actions": [
+		{
+			"uid": "RuleEnableAction",
+			"label": "Enables or disables rules",
+			"description": "This handler can enable or disable a rule or group of rules specified by ID.",
```

Comment:
This should describe the action, not the handler, i.e. the word "handler" should not appear here at all.